valhalla/Valhalla_docker_ubuntu_1804.md

```diff
@@ -129,28 +130,61 @@ mkdir ~/gisops_docker && cd $_
+      - min_y=38 # -> Albania | -90  -> World
+      - max_x=22 # -> Albania |  180 -> World
+      - max_y=43 # -> Albania |  90  -> World
+      - use_tiles_ignore_pbf=True
+      - force_rebuild=False
+      - force_rebuild_elevation=False
+      - build_elevation=True
+      - build_admins=True
+      - build_time_zones=True
 ```
--   `cd` will change the directory in the correct folder for Valhalla.
+-   `volumes` is mounting a local folder called `custom_files` into the docker container. This is the place where the Valhalla container shares its data and also the place where changes to the running Instance can be made.
+-   `tile_urls` will be downloaded by Valhalla and build if their file hashes don't match with the existing build tiles. It's possible to add as many urls as desired with a space delimeter. 
+-   `min_*` defines the bounding box so Valhalla can download the correct elevation tiles. This is only needed if `build_elevation` is set to `True`.
+-   `use_tiles_ignore_pbf` will show Valhalla, that it should prioritize tiles that were already build and not overwrite it.
+-   `force_rebuild` will force Valhalla to rebuild the tiles in all cases except for `force_rebuild_elevation`.
+-   `force_rebuild_elevation` will always skip rebuilding the elevation data if `False`. That way downloading and processing huge amount of elevation data can be omitted.
+-   `build_*` will tell Valhalla what to build in detail. All of them are optional for a functioning Valhalla instance.
+
+
+## 2. Run docker-compose.yml
+
 
-## 2. Run docker-compose
 Now let `docker-compose` build Valhalla by typing:
 ```bash
- docker-compose up -d
+docker-compose up --build
 ```
--   `up` will create and start the Valhalla docker container
--   `-d` runs the container in the background without binding the terminal on it.  
+Docker will now get the latest Valhalla image from the [GIS • OPS Dockerhub](https://hub.docker.com/repository/docker/gisops/valhalla) and build it with the osm file of Albania and Andorra.   
 
+If you check the content of your folder you will notice a new folder named `custom_files`. In there you'll find all the files Valhalla needs to properly run or rebuild any of its data. You can copy, compress, move the folder even to a different system and as long as you use the same `docker-compose.yml`, Valhalla will be able to immediately start up with your data.
 
-The `build time` can be `more than 15 minutes`, depending on your hardware. To decrease the build time, be aware that `the build process will use all of your available cores`, so don't do anything CPU consuming while building Valhalla. Watching a movie should be fine though ;)...  
+**Note**: The `build time` can be `more than 15 minutes`, depending on your hardware. To decrease the build time, be aware that `the build process will use all of your available cores`, so don't do anything CPU consuming while building Valhalla. Watching a movie should be fine though ;)...  
```

Comment:
- `, so don't do anything CPU consuming while building Valhalla. Watching a movie should be fine though ;)...  `